ui/src/components/SampleView/MotorControls.jsx

```diff
@@ -25,21 +25,22 @@ function MotorControls() {
   );
 
   if (!verticalMotorProps || !horizontalMotorProps) {
-    return motorsProps.map(({ attribute: k, role }) => (
-      <MotorInput key={k} role={role} />
-    ));
+    return motorsProps.map(({ role }) => <MotorInput key={role} role={role} />);
```

Comment:
Unrelated: using the `role` as key instead of the attribute to fix React warning about duplicate keys.